### PR TITLE
Remove overrides

### PR DESCRIPTION
**Pull request by pohnemus**
Cleaning up some overrides.

These are actually duplicated in this repository so removed them from the index file:
* 'max-lines-per-function': 'off',	
* 'max-nested-callbacks': 'off',	
* 'max-statements': 'off',	
* 'no-undefined': 'off',	
* 'no-unused-expressions': 'off',	
* 'no-unused-vars': 'off',

This has been deprecated in favor of `padding-line-between-statements`:
* 'newline-after-var': 'off',	

This has also been totally deprecated:
* 'no-process-env': 'off',

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,14 +82,6 @@ module.exports = {
         'jest/valid-expect': 'error',
         'jest/valid-title': 'error',
         'max-lines': 'off',
-        'max-lines-per-function': 'off',
```
Comment (tclindner) [inline index.js:85]: On second thought, we might want to add `max-lines-per-function`, `max-statements`, `max-nested-callbacks`, and `no-undefined ` back in. It is useful in tests to check if something is undefined. It is also useful to nest `describe` and `test` blocks. What do you think?

Comment (pohnemus) [reply index.js:85]: I agree they are useful but they are already set in files in the rules folder which I think covers us - right?

Comment (tclindner) [reply index.js:85]: This section is the overrides section for test files only. If we want the rule disabled for tests we will need to add it here.

Comment (pohnemus) [reply index.js:85]: Ahh I missed that. And that makes sense to add back in sorry!

**Pull request merged**